pages/Transfer.qml

```diff
@@ -1094,7 +1098,11 @@ Rectangle {
         title: qsTr("Please choose a file") + translationManager.emptyString
         onAccepted: {
             console.log(walletManager.urlToLocalPath(importKeyImagesDialog.fileUrl))
-            currentWallet.importKeyImages(walletManager.urlToLocalPath(importKeyImagesDialog.fileUrl));
+            if (currentWallet.importKeyImages(walletManager.urlToLocalPath(importKeyImagesDialog.fileUrl))) {
+                appWindow.showStatusMessage(qsTr("Key images successfully imported to wallet") + translationManager.emptyString, 3);
+            } else {
+                appWindow.showStatusMessage(qsTr("Failed to import key images to wallet") + ": " + currentWallet.errorString + translationManager.emptyString, 5);
```

Comment:
```suggestion
                appWindow.showStatusMessage(qsTr("Failed to import key images to wallet") + ": " + currentWallet.errorString.substring(29) + translationManager.emptyString, 5);
```

Reply:
to avoid this duplicated error message:
![image](https://user-images.githubusercontent.com/45968869/132069021-95165980-fe00-4cdb-ab6b-91dfe2b6f2a3.png)

![image](https://user-images.githubusercontent.com/45968869/132069042-45b97067-4794-4ea5-8467-69decaeb03fb.png)

Reply:
We can't do that. There are different error strings with different lengths. We have to print currentWallet.errorString only, without any translations.